### DISCUSSION TITLE
Sharing & workshoping alchemiscale utility scripts

### DISCUSSION THREAD

**Discussion by IAlibay** [openforcefield/discussions#23] [alchemiscale User Questions]
Hi all,

The alchemiscale client is quite powerful but getting everything done can be quite daunting when starting up. I thought this would be a great place for users to share their scripts and possibly work together to create some refined utilities we can all use.

**Comment by IAlibay** [comment]
Here are some of my initial scripts:

Generating & submitting a TYK2 network using openfe-benchmarks: https://gist.github.com/IAlibay/8edaba70d626ed9b94802d885f460faf

Get the status of all the edges in a network (reads from file a network string): https://gist.github.com/IAlibay/d1c13e7cc9d28f05d104b0dc7c59b103

Print out the tracebacks of all the errors in a network: https://gist.github.com/IAlibay/d2189a92101e434b9752e8198e8065ea

Restart all error-ed edges in a network: https://gist.github.com/IAlibay/e2222851fa00726b5704e9301c1879af

**Comment by dotsdl** [comment]
Thanks for getting this started @IAlibay! 

Seeing how users are making use of the client will give us essential feedback on the usage patterns we should optimize for. In other words, the scripts users write and share here will likely be the precursors to functionality we build into future iterations of the client.

**Comment by IAlibay** [comment]
Here's an netbook for generating, visualising and submiting a transformation network from an SDF file, PDB, and a pre-defined list of edges (labelled by ligand names): https://gist.github.com/IAlibay/29b7947db94aed6166dfb62bf3eca252
